applications/trilinos_application/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h

```diff
@@ -768,6 +766,8 @@ class TrilinosBlockBuilderAndSolver
             if(temp_size <1000) temp_size = 1000;
             int* temp = new int[temp_size]; //
 
+            auto rElements = rModelPart.Elements();
+            auto rConditions = rModelPart.Conditions();
```

Comment:
I think they both should be auto& (otherwise a copy is made).

Reply:
I see the point. Done.